### ISSUE TITLE
Serve images in next-gen formats

### ISSUE THREAD

**Issue by tabshaikh** [internetarchive/openlibrary#3222]
Image formats like JPEG 2000, JPEG XR, and WebP often provide better compression than PNG or JPEG, which means faster downloads and less data consumption. Current .png and .jpg extensions in open library slow the loading of the site significantly almost 1.8s and there is an opportunity here to improve the page load speed.

### Evidence / Screenshot (if possible)
![Screenshot from 2020-03-20 13-07-49](https://user-images.githubusercontent.com/31198893/77144472-00e2bc80-6aac-11ea-9d6b-938d176fa5ea.png)

### Relevant url?

https://developers.google.com/speed/pagespeed/insights/?url=openlibrary.org

### Stakeholders

@tabshaikh

**Comment by BrittanyBunk** [comment]
This doesn't offer much compression, but I wish that I could upload images in BMP and TIFF formats too (I kind of don't like TIFF, so maybe just BMP) - but that might slow the website down.

What if you write that people can upload with a number of file formats, but it's preferable to use the ones you specified to make the site run faster?

**Comment by tfmorris** [comment]
@tabshaikh It would be good to note whether your stats are for Desktop or Mobile, since they have different performance profiles.

This should be a secondary (or tertiary) task after deferring download of offscreen images (#1), reducing size of downloaded images to match rendered size, and fixing cache TTLs (the current 10 min for cover images is crazy short).

**Comment by tabshaikh** [comment]
@tfmorris sure I will update that on all the issues. @tfmorris If you could have a look at this issue here https://github.com/internetarchive/openlibrary/issues/3227 which is an `epic` for all related performance problems there is a list of task seen from the audit and the audit for desktop and mobile is also linked in the description. It would be helpful if you could help me arrange these tasks  based on their priorities :)

**Comment by guyjeangilles** [comment]
@tabshaikh I'm assuming the plan is to do this for more than just the homepage?

**Comment by tabshaikh** [comment]
@guyjeangilles yes will update all the issues for the author and edition pages too :)
